documentai/snippets/quickstart.go

```diff
@@ -0,0 +1,73 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START documentai_quickstart]
+
+// quickstart sends a file at a given filePath for online processing.
+package snippets
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"log"
+
+	documentai "cloud.google.com/go/documentai/apiv1"
+	"cloud.google.com/go/documentai/apiv1/documentaipb"
+	"google.golang.org/api/option"
+)
+
+func quickstart() {
+
+	projectID := "PROJECT_ID"
+	locationID := "us"
+	processorID := "aaaaaaaa" // Create a Processor before running sample
+	filePath := "invoice.pdf"
+	mimeType := "application/pdf"
+
+	ctx := context.Background()
+
+	endpoint := fmt.Sprintf("%s-documentai.googleapis.com:443", locationID)
```

Comment:
Don't switch between `fmt` and `log` in the same sample. Pick one and stick with it.

Reply:
log is only used for error reporting. All of the other samples I looked at followed this pattern

Reply:
nvm, found this in the contributing guide https://github.com/GoogleCloudPlatform/golang-samples/blob/main/CONTRIBUTING.md#return-errors